File: ws.chatengine.io/src/lib/redis.js
import { Redis } from "ioredis";

export const redisCache = new Redis({
  host: process.env.REDIS_HOST,
  port: process.env.REDIS_PORT,
  db: 0, // 0 for caching
});

export const redisSubscriber = new Redis({
  host: process.env.REDIS_HOST,
  port: process.env.REDIS_PORT,
  db: 1, // 1 for pub/sub
});

File: api.chatengine.io/server/redis.py
import redis
from server import settings

# Create a Redis pubsub connection
redis_pubsub = redis.Redis(
    host=settings.REDIS_HOST, 
    port=settings.REDIS_PORT, 
    db=1
)

# Create a Redis cache connection
redis_cache = redis.Redis(
    host=settings.REDIS_HOST, 
    port=settings.REDIS_PORT, 
    db=0
)
